packages/jitm/src/Manager.php

```diff
@@ -40,8 +44,8 @@ static function init() {
 	/**
 	 * Jetpack_JITM constructor.
 	 */
-	private function __construct() {
-		if ( ! Jetpack::is_active() || Jetpack::is_development_mode() ) {
+	public function __construct() {
+		if ( ! \Jetpack::is_active() || \Jetpack::is_development_mode() ) {
```

Comment:
It would be good if this was a filter maybe? That we standardize? Or just use the current `jetpack_just_in_time_msgs` to disable when Jetpack is not active or is in  development mode.